bits/sendgrid/sendgrid.go

```diff
@@ -0,0 +1,106 @@
+package sendgrid
+
+import (
+	"context"
+	"net/http"
+
+	"encore.dev"
+	"encore.dev/beta/errs"
+	"encore.dev/pubsub"
+	"encore.dev/rlog"
+	"github.com/sendgrid/sendgrid-go"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+)
+
+// Encore docs about secrets: https://encore.dev/docs/primitives/secrets
+var secrets struct {
+	SendGridAPIKey string
+}
+
+type SendParams struct {
+	From    string `json:"form"` // sender email address
+	To      string `json:"to"`   // recipient email address
```

Comment:
You removed the ability to set the sender/receiver name; how come?

Reply:
Please tell me in what form you would like to see this structure?

Apparently I just didn't fully understand

Reply:
I was thinking something like this:
```go
type Address struct {
	Name string `json:"name"`
	Email string `json:"email"`
}

type SendParams struct {
	From Address `json:"from"` // sender email address
	To   Address `json:"to"`   // recipient email address
	// ...
}
```

Reply:
oh i got you, thanks!
pushed
269d877ce1261b8f31a576f23007aacb9589dce5